backend/corpora/common/corpora_config.py

```diff
@@ -68,3 +68,10 @@ def get_defaults_template(self):
             template["issuer"].append(self.api_signin_url)
 
         return template
+
+
+class CorporaCloudfrontConfig(SecretConfig):
+    environ_source = "CORPORA_CLOUDFRONT_CONFIG"
+
+    def __init__(self, *args, **kwargs):
+        super().__init__("backend", secret_name="cloudfront", **kwargs)
```

Comment:
I think you need to add a new line to here and in the cloudfront file to pass the linter